### DISCUSSION TITLE
named routes in Controller

### DISCUSSION THREAD

**Discussion by 42pe** [adonisjs/core#2043] [Help] [answered]
How can I used the route helper in a Controller?

e.g, if I want to redirect the user to the login page after registration?

using:
`response.redirect(route('login'))`
renders an error: "route is not defined "

**Comment by thetutlage** [comment, marked as answer]
Which version of the framework ?

**Reply by 42pe** [reply]
Sorry! v4.1.0, fullstack-app boilerplate, if that matters.

**Reply by ammezie** [reply]
It should be
`response.route('login')`

**Reply by 42pe** [reply]
Thanks! that works!
